xcdat/regridder/base.py

```diff
@@ -10,20 +10,20 @@
 
 
 def preserve_bounds(
-    source: xr.Dataset, source_grid: xr.Dataset, target: xr.Dataset
+    source: xr.Dataset, target_grid: xr.Dataset, target: xr.Dataset
 ) -> xr.Dataset:
     """Preserves bounds from sources to target.
 
-    Preserve the lat/lon bounds from `source_grid` to `target`.
+    Ensure the lat/lon bounds from `target_grid` are included in the `target` dataset.
 
     Preserve any additional bounds e.g. time, vertical from `source` to `target`.
 
     Parameters
     ----------
     source : xr.Dataset
         Source Dataset.
-    source_grid : xr.Dataset
-        Source grid Dataset.
+    target_grid : xr.Dataset
+        Target grid Dataset.
```

Comment:
Did I rename things correctly here? I think we are passing in the target grid here (even though it is initially being used to "source" some bounds information).